mlrun/data_types/spark.py

```diff
@@ -125,6 +125,22 @@ def get_df_stats_spark(df, options, num_bins=20, sample_size=None):
     summary_df.set_index(["summary"], drop=True, inplace=True)
     if summary_renamed:
         summary_df.rename(columns={"__summary_internal__": "summary"}, inplace=True)
+    # pandas df.describe() returns std, while spark returns stddev
+    # we therefore need to rename stddev to std for compatibility with pandas
+    # we may want to consider going with stddev in 1.2 and beyond
```

Comment:
```suggestion
    # TODO: consider going with stddev in 1.2 and beyond
```

This will make Pycharm (at least) add this to the TODO list, which will make it easier to track this.